### ISSUE TITLE
Addon not showing dragonring vigor charges

### ISSUE THREAD

**Issue by paessz** [tullamods/Dominos#875]
version 11.0.0 retail

The addon dont show dragonriding vigor charges anymore just if I reload the game in a dragonriding mount

**Comment by Tuller** [comment]
Do you have the encounter bar visible?

**Comment by paessz** [comment]
Yes, I have

**Comment by alliepet** [comment]
I'm experiencing the same problem. The Encounter Bar is enabled in Edit Mode. Right now Im using an old Dominos version (pre 11.0). When I update Dominos the skyriding vigor bar is no longer there. Double-checked edit mode and all that too.

**Comment by ANaPhOrAX** [comment]
yep, it's fucky for me as well. dominos encounter bar is just never showing up at all. and opacity is set to 100. the blizzard encounter bar also is bugging out. like it doesn't initially show, but if I open edit mode, it will show up. so I'm just gonna hide both and get a vigor bar I guess :(
